### PR TITLE
Revert changes to DISPLAY_NAME for CFBundleDisplayName

### PR DESCRIPTION
**Pull request by sleroux**

### CODE REVIEW

File: templates/firefox-ios.xliff [hunk 1]
```diff
@@ -2155,11 +2155,11 @@
   </file>
   <file original="Extensions/SendTo/Info.plist" source-language="en" datatype="plaintext">
     <header>
-      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3" build-num="8W109m"/>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.2" build-num="8C38"/>
     </header>
     <body>
       <trans-unit id="CFBundleDisplayName">
-        <source>$(DISPLAY_NAME)</source>
+        <source>Send Tab</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff]: Would add a localization note here

File: templates/firefox-ios.xliff [hunk 2]
```diff
@@ -2241,11 +2241,11 @@
   </file>
   <file original="Extensions/ViewLater/Info.plist" source-language="en" datatype="plaintext">
     <header>
-      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3" build-num="8W109m"/>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.2" build-num="8C38"/>
     </header>
     <body>
       <trans-unit id="CFBundleDisplayName">
-        <source>$(DISPLAY_NAME)</source>
+        <source>View Later</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff]: Would add a localization note here too

File: templates/firefox-ios.xliff (hunk 1 above)
Comment (flodolo) [inline templates/firefox-ios.xliff]: IIRC it's technically not possible for these strings (CFBundle*)

Comment (flodolo) [inline templates/firefox-ios.xliff]: OK, I was remembering correctly https://bugzilla.mozilla.org/show_bug.cgi?id=1277515#c9

Comment (Delphine) [inline templates/firefox-ios.xliff]: Aha, I see :) thanks!

**Pull request merged**